awscli/customizations/sso/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
For reviewers, I lifted this from the IDE integration.

I did not include some of the extra styles and fonts, since having everything simplifies serving the resources from the local server. But let me know if you'd like the additional fonts/icons.

Reply:
Unrelated to the above from an internal discussion, we need to make sure that this file is included in the source distribution. DOes it need to be in `manifest.in`?

Reply:
It shouldn't need to be since it's under the top-level package directory (`awscli`). We can generate the sdist to double check. Also I don't think v2 will use a `manifest.in`, the sdist file inclusions outside of the core package should be listed [here](https://github.com/aws/aws-cli/blob/07dee1855177584cd82d9016254d46b046d095ab/pyproject.toml#L54-L65).